tests/BuyAdapterFuzz.t.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC20Detailed} from '@aave/core-v3/contracts/dependencies/openzeppelin/contracts/IERC20Detailed.sol';
+import {IPoolAddressesProvider} from '@aave/core-v3/contracts/interfaces/IPoolAddressesProvider.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AugustusRegistry} from 'src/lib/AugustusRegistry.sol';
+import {MockParaSwapBuyAdapter} from './mocks/MockParaSwapBuyAdapter.sol';
+import {BaseTest} from './utils/BaseTest.sol';
+
+contract BuyAdapterFuzzTest is BaseTest {
+  MockParaSwapBuyAdapter internal buyAdapter;
+  address[] internal aaveV3EthereumAssets;
+
+  function setUp() public override {
+    super.setUp();
+    vm.createSelectFork(vm.rpcUrl('mainnet'));
+
+    buyAdapter = new MockParaSwapBuyAdapter(
+      IPoolAddressesProvider(address(AaveV3Ethereum.POOL_ADDRESSES_PROVIDER)),
+      address(AaveV3Ethereum.POOL),
+      AugustusRegistry.ETHEREUM
+    );
+    aaveV3EthereumAssets = [
```

Comment:
can we use the `getReserves` function of the pool so we are not limited to only 3 assets?